Casks/w/wins.rb

```diff
@@ -0,0 +1,20 @@
+cask "wins" do
+  version :latest
+  sha256 :no_check
```

Comment:
Replace with the actual value.
```suggestion
  sha256 :no_check
```